tests/suite/cli_paths.rs

```diff
@@ -389,7 +391,9 @@ mod windows {
         let mut cx = CliTestContext::new(Scenario::Empty).await;
         let _guard = RegistryGuard::new(&USER_PATH).unwrap();
         let cfg_path = cx.config.cargodir.join("bin").display().to_string();
-        let get_path_ = || get_path().unwrap().unwrap().to_string();
+        let path = get_path().unwrap().unwrap();
+        let path = unsafe { from_raw_parts(path.as_ptr() as *const u16, path.len() / 2) };
```

Comment:
These two lines of code was partially peeked from [`winreg`](https://github.com/gentoo90/winreg-rs/blob/1c56127adf4e310b5c8ed2af89aa3f7b09d38371/src/types.rs#L34-L38).

Reply:
There must be a nicer way of doing this...

Reply:
@InfyniteHeap There's nothing wrong in bypassing nightly-exclusive APIs with unsafe, but adding a `TODO` comment will definitely help here :)

Reply:
@InfyniteHeap Now that https://github.com/microsoft/windows-rs/pull/3190 is online, I think you should explore using the `HSTRING` type rather than using bytestrings directly.

Reply:
Yes, there should be no reason to use `unsafe` for any registry value conversion.